<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Administrators-MacBook-Pro-9.local" ignored="0" name="RetryMulitpleFailedTestCases" tests="11" failures="2" timestamp="04 Dec 2017 17:03:07 GMT" time="0.086" errors="0">
  <testcase name="Test1" time="0.019" classname="mytests.MyTests">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.001" classname="mytests.MyTests">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.002" classname="mytests.MyTests">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.001" classname="mytests.MyTests">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.002" classname="mytests.MyTests">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at mytests.MyTests.Test1(MyTests.java:12)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase name="Test2" time="0.001" classname="mytests.MyTests">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.002" classname="mytests.MyTests">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.001" classname="mytests.MyTests">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.001" classname="mytests.MyTests">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.0" classname="mytests.MyTests">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at mytests.MyTests.Test2(MyTests.java:18)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
  <testcase name="Test3" time="0.001" classname="mytests.MyTests"/>
</testsuite> <!-- RetryMulitpleFailedTestCases -->
